extensions/amp-mplayer/0.1/amp-mplayer.js

```diff
@@ -0,0 +1,480 @@
+/**
```

Comment:
There are bunch of lint error here. Use `gulp lint --local-changes --fix` to fix them. Also if you use Visual Studio Code, donwload "prettier" plugin. We use prettier for code formatting

Reply:
I ran it and than the lint test couple of times and got:
SUCCESS: No linter warnings or errors.
Still, the code is not running. I get the following error:

/Users/ofirshlifer/RubymineProjects/amphtml/build/video-autoplay.css:16
i-amphtml-video-mask {
                     ^
ParseError: Unexpected token

Reply:
Could you please try:
1- rebase with latest master
2- run `gulp clean`
3- run `yarn`
4- run `gulp`

Reply:
OK now it works, thanks (:
I still have a problem with the check-types test. Im getting the following error:

ERROR: Could not start nailgun-server.jar on port 2114...
The following tasks did not complete: check-types
Did you forget to signal async completion?

Cant figure out why am I getting this.

Reply:
Are you running on Windows? I'm not entirely sure why it's failing on your local machine.

Either way, part of the checks on Travis show that you need to add your extension to the `video-manager` dependency whitelist:

https://github.com/ampproject/amphtml/blob/aea52cd53de791261fcffeb071912bf66053aa47/build-system/dep-check-config.js#L286-L305

Reply:
No, Im running on mac. 
I added the extension to the whitelist. Is there another file I should add our extension to?
Regarding the test - is it a problem that the test is failing on my machine? Is it running on yours?

Reply:
The checks are running locally on my computer and it seems like the tests are failing here but when I look on the tests results I see old versions of our project. How can it be?

Furthermore, do I need to make changes on the validator-amp-minute-media-player.out file? 
Or the validator-amp-minute-media-player.html file?

Reply:
Run `gulp validator --update_tests` to get new `.out` files.